release-notes/8.0/known-issues.md

```diff
@@ -4,14 +4,29 @@ You may encounter the following known issues, which may include workarounds, mit
 
 ## .NET SDK
 
-### [8.0.100-preview.7] .NET 8 preview 7 SDK is not compatible with Visual Studio 17.6
+### [8.0.100-rc.2] .NET 8 RC 2 SDK is not compatible with Visual Studio 17.7 or 17.8-preview 2 for Razor projects
 
-Please upgrade to 17.7.0 or 17.8-preview1 or use [global.json](https://learn.microsoft.com/dotnet/core/tools/global-json) to pin to the 7.0.400 SDK that is included with 17.6.
+You will see errors like the following when building Razor applications in incompatible Visual Studio versions when using .NET 8 RC 2 SDK
+```
+System.TypeLoadException: Could not load type 'Enumerator' from assembly 'Microsoft.CodeAnalysis, Version=4.8.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. 
+ C:\Program Files\Microsoft Visual Studio\2022\Preview\MSBuild\Current\Bin\Roslyn\Microsoft.CSharp.Core.targets 84
+```
+**Workaround** Please upgrade to 17.8-preview3 or use [global.json](https://learn.microsoft.com/dotnet/core/tools/global-json) to pin to the 7.0.400 SDK that is included with 17.7.
+
+### [8.0.100-rc.2] .NET 8 RC 2 SDK will list out 14 warnings when running dotnet new the first time
```

Comment:
I'd probably use "display" instead of "list out". Also, it doesn't *directly* apply to `dotnet new`. It applies to installation of project templates. The first use of `dotnet new` will install project templates. But any call to `dotnet new install` against any project template will also show the warnings.